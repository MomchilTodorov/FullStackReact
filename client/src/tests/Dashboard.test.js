import React from "react";
import { shallow, configure } from "enzyme";
import Dashboard from "../components/Dashboard";
import Adapter from "enzyme-adapter-react-16";

configure({ adapter: new Adapter() });

describe("Dashbaord", () => {
  it("should render Dashboard", () => {
    const component = shallow(<Dashboard />);
    expect(component.getElements()).toMatchSnapshot();
  });
});
