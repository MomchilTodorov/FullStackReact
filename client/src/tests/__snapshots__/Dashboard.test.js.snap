// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashbaord should render Dashboard 1`] = `
Array [
  <div>
    <Connect(SurveyList) />
    <div
      className="fixed-action-btn"
    >
      <Link
        className="btn-floating btn-large blue-grey darken-4"
        replace={false}
        to="/surveys/new"
      >
        <i
          className="material-icons"
        >
          add
        </i>
      </Link>
    </div>
  </div>,
]
`;

exports[`MyComponent should render initial layout 1`] = `
Array [
<div>
    <input
         type="text"
    />
</div>,
]
`;
